handlers/whatsapp_legacy/handler.go

```diff
@@ -733,12 +733,25 @@ func buildPayloads(msg courier.MsgOut, h *handler, clog *courier.ChannelLog) ([]
 			payload.Template.Language.Policy = "deterministic"
 			payload.Template.Language.Code = langCode
 
-			component := &Component{Type: "body"}
+			for k, v := range templating.Params {
+				if k == "body" {
+					component := &Component{Type: "body"}
```

Comment:
Just body to keep the same features but no new components supported for legacy WA channels